### ISSUE TITLE
Required fields lack aria-required="true"

### ISSUE THREAD

**Issue by gcamacho079** [formkit/formkit#984]
### Reproduction

https://formkit.link/a2aa83e7a381f0490d76639b7700d982

### Describe the bug

When I used axe DevTools to test FormKit fields, it raised the issue that required fields are missing the `aria-required` attribute.

According to [Deque’s Issue Help documentation](https://docs.deque.com/issue-help/1.0.0/en/aria-required-missing), in addition to being marked visually as required, fields should also use `aria-required="true"` to programmatically indicate that the field is required.

This also helps screen reader users when developers fail to add visual or text indication in the label that it’s required, as the screen reader will still announce that it's required.

### Environment

• OS: iOS
• Browser: Chrome
• Version: ?

**Comment by justin-schroeder** [comment]
We’ll get on this asap 👍

**Comment by justin-schroeder** [comment]
We did this in 1.4.0 🎉

**Comment by gcamacho079** [comment]
@justin-schroeder thank you!
